app/Action/Action.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace app\Action;
+
+abstract class Action
+{
+    const ACTION_COMPLETED = 'action_completed'; // выполнить
```

Comment:
```suggestion
    /** @var string выполнить */
    const ACTION_COMPLETED = 'action_completed';
```
Некритично, но раз уж ты пишешь комментарий к действию, то рекомендую такие комментарии делать в phpdoc, чтобы можно было потом в любом месте узнать что это за константа через быстрые подсказки в IDE